build.gradle

```diff
@@ -337,7 +337,7 @@ configurations.all {
 }
 
 dependencies {
-  implementation 'com.github.hmcts:civil-commons:1.0.90'
+  implementation 'com.github.hmcts:civil-commons:v1.0.90-CIV-13860-2'
```

Comment:
The commons branch needs to be released before this is merged and then this line will be updated with latest commons version.